src/System.Data.SqlClient/src/System.Data.SqlClient.csproj

```diff
@@ -5,7 +5,7 @@
     <ProjectGuid>{D4550556-4745-457F-BA8F-3EBF3836D6B4}</ProjectGuid>
     <AssemblyName>System.Data.SqlClient</AssemblyName>
     <AllowUnsafeBlocks>true</AllowUnsafeBlocks>
-    <IsPartialFacadeAssembly Condition="'$(TargetGroup)' == 'net463'">true</IsPartialFacadeAssembly>
+    <IsPartialFacadeAssembly Condition="'$(TargetGroup)' == 'netfx'">true</IsPartialFacadeAssembly>
```

Comment:
Remove this line. It wasn't working already because targetGroup was never net463 and I checked and there are no netfx configurations for this project, so we should remove this.